test/jdk/java/foreign/TestLibraryLookup.java

```diff
@@ -100,10 +100,20 @@ public void testVariableSymbolLookup() throws Throwable {
         waitUnload();
     }
 
-    @Test(expectedExceptions = IllegalArgumentException.class)
-    public void testBarVariableSymbolLookup() throws Throwable {
+    @Test
+    public void testBadVariableSymbolLookup() {
         LibraryLookup lookup = LibraryLookup.ofLibrary("LookupTest");
-        lookup.lookup("c", MemoryLayouts.JAVA_INT.withBitAlignment(1 << 16)).get();
+        try {
+            MemoryLayout layout = MemoryLayouts.JAVA_INT.withBitAlignment(1 << 16);
+            MemorySegment segment = lookup.lookup("c", layout).get();
+            // no exception, check that address is aligned
+            if ((segment.address().toRawLongValue() % layout.byteAlignment()) != 0) {
+                fail("Unaligned address");
+            }
```

Comment:
maybe just use assertFalse here?

Reply:
Right, the behavior is the same, assertFalse is maybe a little more idiomatic, so I thought not using it might have been an oversight. (I'm just nitpicking though. Feel free to ignore)